static/app/components/tabs/tab.tsx

```diff
@@ -82,23 +82,15 @@ export const BaseTab = forwardRef(
     );
 
     return (
-      <TabWrap
+      <NewTabWrap
         {...mergeProps(tabProps, additionalProps)}
         hidden={hidden}
         selected={isSelected}
         overflowing={overflowing}
         ref={ref}
       >
-        <InnerWrap>
-          <StyledInteractionStateLayer
-            orientation={orientation}
-            higherOpacity={isSelected}
-          />
-          <FocusLayer orientation={orientation} />
-          {props.children}
-          <TabSelectionIndicator orientation={orientation} selected={isSelected} />
-        </InnerWrap>
-      </TabWrap>
+        {props.children}
+      </NewTabWrap>
```

Comment:
For sake of getting feedback sooner rather than later, I have edited the BaseTab component directly, but I will decouple the style changes I made here just for draggable tabs from the current BaseTab styles before this PR merges.